Rust/src/lib.rs

```diff
@@ -28,15 +28,20 @@
 //!     .collect();
 //! ```
 //!
+//! ## Scoring Algorithms
+//!
+//! The [`algorithm`] module contains tools and components for creating new
```

Comment:
algorithm seems like it could refer to a lot in the RCF library. Why not call the module scoring or scorer?

Reply:
I went with `algorithm` since forecasting and KNN are not exactly producing "scores". However, I didn't spend all that much time thinking of a name other than "algorithm". Given my comment about scores I'm open to other suggestions. Good names are half the battle.


I see, though, that I write "Scoring Algorithms" in this section of the documentation. I will make the wording more consistent.

Reply:
I would especially like @jotok 's opinion on names. My thought is that the `algorithm` (or whatever) submodule contains all the "good stuff" for researchers.

Reply:
I'm sympathetic Chris's point of view, because in my mind I think of RCF as a data structure on which different scoring algorithms are implemented, but I think Jack's right that this will probably be a point of confusion to new users of the library. What about `visitors.rs`? Barring that, I think some version of "scoring" is fine, if we take an expansive view of what a "score" can be.

Reply:
I like `visitors` as a submodule; `scoring` has the risk of making a narrow first impression of what these things mean. My only issue is that the visitor is only one component of an algorithm implementation. However, I'm still fuzzy on this distinction so for now a visitors sub-module is sufficient.